Basic.Azure.Storage/Communications/BlobService/BlobOperations/GetBlobResponse.cs

```diff
@@ -48,25 +45,25 @@ private DateTime ParseDate(string headerValue)
             DateTime.TryParse(headerValue, out dateValue);
             return dateValue;
         }
-        
-        public async Task ParseResponseBodyAsync(System.IO.Stream responseStream)
+
+        public virtual async Task ParseResponseBodyAsync(System.IO.Stream responseStream)
         {
             _stream = responseStream;
```

Comment:
Eventually, it might be nice to preserve this data as either a MemoryStream or a byte[]. That will allow GetDataBytes() and GetDataStream() to be indempotent instead of one-off, as they are today.